kitsune/wiki/management/commands/send_weekly_ready_for_review_digest.py

```diff
@@ -47,19 +47,21 @@ def _send_mail(locale, user, context):
         products = Product.objects.all()
 
         for loc in locales:
-            docs = revs.filter(document__locale=loc).values_list("document", flat=True).distinct()
-            docs = Document.objects.filter(id__in=docs)
+            doc_ids = (
```

Comment:
Obviously this query is not introduced now, but we definitely have `N+1` queries here. I am wondering if there's away to reshape this part to avoid that problem

Reply:
After further investigation, I don't think we can eliminate the `N+1` queries here, because the locales determine the users, and each user could possibly see a different set of visible docs.